src/aioswitcher/device/tools.py

```diff
@@ -211,3 +213,24 @@ def validate_token(username: str, token: str) -> bool:
     else:
         logger.debug("request failed with status code: %s", response.status_code)
     return is_token_valid
+
+
+def get_shutter_index(device_type: DeviceType, device_num: int) -> int:
+    """Return the currect shutter index (based of device type)."""
+    if device_type == DeviceType.RUNNER or device_type == DeviceType.RUNNER_MINI:
+        if device_num == 1:
+            return 1
+    if device_type == DeviceType.RUNNER_S11:
+        if device_num == 1:
+            return 3
+    return 0
+
+
+def get_light_index(device_type: DeviceType, device_num: int) -> int:
+    """Return the currect light index (based of device type)."""
```

Comment:
Please explain this.

Reply:
I will explain.
get_shutter_index is used to get the correct position to look for in the packet for getting the shutter's direction/position.
get_light_index is used to get the correct position to look for in the packet for getting the light status if it's on or off.

The main idea of the position is like this:

- RUNNER/RUNNER MINI:
The light index does not exist
The shutter index is 0
- Runner S11:
The first light index is 0
The second light index is 1
The shutter index is 2
- Runner S12:
The light index is 0
The first shutter index is 1
The second shutter index is 2

So basically, if the device has lights, it comes before the shutter.

I could remove get_shutter_index and get_light_index altogether and put these hardcoded values inside the bridge.py file.
I just figured it cleaner to look on this part like this.